shared/haml/course_explorer_table.haml

```diff
@@ -199,12 +251,13 @@
               .arrow
                 %i.arrow-down{class: "fa fa-caret-down"}
                 %i.arrow-up{class: "fa fa-caret-up"}
-          - if course_index == index_course_first_same_line
-          - elsif course_index == index_course_first_same_line + 1
-            = detail_containers[index_course_first_same_line]
-            = detail_containers[index_course_first_same_line + 1]
-          - else
-            = detail_containers[course_index]
+          %div{style: "width: 935px;"}
```

Comment:
where does this width come from?

Reply:
Chatted elsewhere. This is the width of the course explorer. Probably out of scope to change in this PR, but we should make this 970 eventually.